### ISSUE TITLE
Update action ignores expand

### ISSUE THREAD

**Issue by Sikian** [sphereio/sphere-node-sdk#145]
### Expected behavior
Using the .expand() modifier should send expand=... as part of the query for creating, updating, querying and deleting a number of resources. 

### Actual behavior
As far as I've been able to see, the BaseService's update method uses the _save method, which doesn't take into consideration anything else than the endpoint. Fiddling with the library to send the expand parameters in the query returns a successfully expanded object.

### Steps to reproduce the behavior
Perform an update query using the expand modifier. A good example could be updating a cart with a new line item and expand to get the productType of the lineItems.
```
SphereClient.getClient().carts.byId(cartId).expand('lineItems[*].productType.name').update({
   version: version,
   actions: [action]
});
```

The result of this query will be the same as without the expand.

**Comment by emmenko** [comment]
@Sikian thanks for reporting this. Indeed reference expansion is not enabled for POST requests. We'll fix this as soon as possible.

**Comment by Sikian** [comment]
@emmenko Thanks for the fast response! Looking forward for the fix

**Comment by emmenko** [comment]
@Sikian this is now included in the [`v1.15.0`](https://github.com/sphereio/sphere-node-sdk/releases/tag/v1.15.0) release
